test/addons/submariner/test

```diff
@@ -128,7 +128,7 @@ def wait_for_service_export(cluster, namespace):
     print(f"Waiting for service in namespace '{namespace}' in cluster '{cluster}'")
     kubectl.wait(
         f"serviceexports/{SERVICE}",
-        "--for=condition=Synced=True",
+        "--for=condition=Ready",
```

Comment:
This change is related to the version change?

Reply:
Yes, see the linked submariner issue.